### ISSUE TITLE
Confusing about the pivot gemfire this example connects to?

### ISSUE THREAD

**Issue by steeeeets** [spring-guides/gs-accessing-data-gemfire#9]
Hi,

I this example, I haven't found any locator, query server start up. I see it says, the region is "LOCAL"?
Local here means the region type of gemfire?
How to understand this local?
I think pivot gemfire is a commerical software, people pay for it, but in this example, seems there an embedded one in spring-data-jap?

Please help. Thanks.

**Comment by jxblum** [comment]
Yes, this is a self-contained, [Pivotal GemFire](https://pivotal.io/pivotal-gemfire) (or alternatively, [Apache Geode](http://geode.apache.org/), if you prefer) cache client (i.e. `org.apache.geode.cache.client.ClientCache`) application.

> NOTE: Apache Geode is the open source core of Pivotal GemFire. As of SD Kay, the dependencies (i.e. `org.springframework.data:spring-data-geode` & `org.springframework.data:spring-data-gemfire`) are interchangeable.  See [here](https://spring.io/blog/2017/10/26/diff-q-spring-data-gemfire-spring-data-geode) for more details.

With Apache Geode, or Pivotal GemFire, a cache client application does not require a server (cluster) to connect and put or retrieve data.  A `ClientCache` instance can, in fact, store data locally using a LOCAL Region data management policy.  This is often useful during development to get you started quickly and easily.

However, as your application requirements and Use Cases (UC) grow, it is very common (and especially recommended in production deployments) to switch to a [client/server](http://geode.apache.org/docs/guide/17/topologies_and_comm/cs_configuration/chapter_overview.html) topology.  That is, 1 or more `ClientCache`, _Spring Boot_ (using _Spring Data for Apache Geode or Pivotal GemFire_) applications connect to a dedicated cluster of Geode/GemFire servers managing the data, bound together through 1 or more Locators.  Clients can connect directly to 1 or more `CacheServers`, but more likely than not, you want to configure your client `Pool` to connect to 1 or more Locators in the cluster of servers to take advantage of single-hop, load-balanced and automatic failure capabilities of the client connection Pool.

But, to get started, it is far less overhead (e.g. faster) and simple to standup a local cache client.  Then, you can progress towards a client/server topology and scale-out your cluster as your demand dictates.

You can read more about [Region Data Storage and Distribution](http://gemfire.docs.pivotal.io/95/geode/developing/region_options/chapter_overview.html) in the User Guide.

This is the [exact definition](http://gemfire-95-javadocs.docs.pivotal.io/org/apache/geode/cache/client/ClientRegionShortcut.html#LOCAL) of a LOCAL Region.

[Pivotal GemFire](https://pivotal.io/pivotal-gemfire) is commercial software, owned and sold by Pivotal Software, Inc.  However, you can start with Open Source, with [Apache Geode](http://geode.apache.org/), and migrate to the commercial offering as you need.  It is also possible to move from commercial back to Open Source (i.e. by switching from _Spring Data for Pivotal GemFire_ back to _Spring Data for Apache Geode_; going both ways is possible as explained in my [blog entry](https://spring.io/blog/2017/10/26/diff-q-spring-data-gemfire-spring-data-geode)).

Now, Pivotal even offers the [Pivotal Cloud Cache](https://docs.pivotal.io/p-cloud-cache/1-5/index.html) service (PCC) on Pivotal CloudFoundry (PCF), which is built on Pivotal GemFire, which is built on Apache Geode.  

Therefore, when you use the new [Spring Boot for Apache Geode/Pivotal GemFire](https://github.com/spring-projects/spring-boot-data-geode) project (documentation available [here](https://github.com/spring-projects/spring-boot-data-geode#documentation)), you are covered "locally" all the way to your journey to the cloud.

See my S1P-2018 [talk](https://springoneplatform.io/2018/sessions/scaling-spring-boot-applications-in-real-time) ([video link](https://www.youtube.com/watch?v=Wsl2fLNM5Q4)) this year.

Hope this helps!
